apps/src/common-styles.module.scss

```diff
@@ -0,0 +1,53 @@
+// As we move from inline css to css modules we are temporarily duplicating commonStyles.js into this css module.
+// Any changes in one should be made in the other to apply to all components.
+@use "color.scss";
+@use "style-constants.scss";
+
+.hidden {
+  display: none;
+}
+
+.purpleHeader {
+  height: style-constants.$workspace-headers-height;
+  background-color: color.$purple;
+  color: color.$white;
```

Comment:
Just curious, why do we need to use reference these explicitly with `color.`? Is it necessary when using `@use`?

Reply:
yep, it was necessary when I switched to `@use`